applications/matter_bridge/sample.yaml

```diff
@@ -3,56 +3,70 @@ sample:
   name: Matter Bridge application
 tests:
   applications.matter_bridge.release:
+    sysbuild: true
```

Comment:
Does it mean we are going to stop testing old solution at all? I think that we are not going to stop supporting it with this release, we should keep testing both solutions (at least some basic smoke tests). @doublemis1 fyi.

Reply:
there are no child/parent image builds left in this but some do need to be added to ensure nothing breaks but this will only be a couple of projects (still tbd)

Reply:
If the sysbuild includes all necessary features which will enabled building our solution in all configuration then I will move   directly to sysbuild, however create additional configuration per platform without sysbuild to validate previous (still supported) approach:
* Template sample for all supported major platform
* Matter Bridge App for supported platform
* Matter Weather Station App for supported platform